src/UnitTests/test_quote_prov.cpp

```diff
@@ -378,16 +378,21 @@ constexpr auto CURL_TOLERANCE = 0.04;
 void RunQuoteProviderTests(bool caching_enabled=true)
 {
     std::clock_t start;
-    double duration_curl;
-    double duration_local;
+    double duration_curl_cert;
+    double duration_local_cert;
+    double duration_curl_verification;
+    double duration_local_verification;
     local_cache_clear();
 
     start = std::clock();
     GetCertsTest();
-    duration_curl = (std::clock() - start) / (double)CLOCKS_PER_SEC;
+    duration_curl_cert = (std::clock() - start) / (double)CLOCKS_PER_SEC;

```

Comment:
std::clock isn't a very good approximation for a wall clock (which is presumably what you are looking for here). I would suggest std::chrono::system_clock instead (which produces time_points, from which you can get typed durations etc).

Reply:
Thanks! I cleaned this up a bit and am using std::chrono::system_clock